### PR TITLE
Accordion default slot

### PR DESCRIPTION
**Pull request by m4thieulavoie**
# Pull Request

## 📖 Description

This PR addresses the issue in accordion where it's unclear whether the slot name is `"item"` or `""`. We now default to a default slot, while maintaining backward compatibility.
Fixes https://github.com/microsoft/fast/issues/5437

### 🎫 Issues

https://github.com/microsoft/fast/issues/5437

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [x] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts [hunk 1]
```diff
@@ -15,10 +15,7 @@ export const accordionItemTemplate: (
     context: ElementDefinitionContext,
     definition: AccordionItemOptions
 ) => html`
-    <template
-        class="${x => (x.expanded ? "expanded" : "")}"
-        slot="item"
```
Comment (m4thieulavoie) [inline packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts:20]: Turns out FAST was automatically adding the slot under the hood. I assume only a tiny portion of the consumers will have to keep their eyes open for a breaking change, as long as they use a component extending `AccordionItem`

File: packages/web-components/fast-foundation/src/accordion/accordion.template.ts [hunk 2]
```diff
@@ -14,8 +14,9 @@ export const accordionTemplate: (
 ) => ViewTemplate<Accordion> = (
     context: ElementDefinitionContext,
     definition: FoundationElementDefinition
-) => html`
+) => /* TODO: deprecate slot name `item` to only support default slot https://github.com/microsoft/fast/issues/XXXX */ html`
```
Comment (m4thieulavoie) [inline packages/web-components/fast-foundation/src/accordion/accordion.template.ts:17]: @chrisdholt did you want to create the issue, or do you want me to, and you would add any additional details afterwards?

Comment (chrisdholt) [reply packages/web-components/fast-foundation/src/accordion/accordion.template.ts:17]: Would you mind taking care of this? Thank you!

**Review by chrisdholt — APPROVED** [review]
Thanks @m4thieulavoie for this!

File: packages/web-components/fast-foundation/src/accordion/accordion.template.ts (hunk 2 above)
Comment (m4thieulavoie) [reply packages/web-components/fast-foundation/src/accordion/accordion.template.ts]: Done!

**Review by EisenbergEffect — APPROVED** [review]

**Review by eljefe223 — APPROVED** [review]

**Pull request merged**